### ISSUE TITLE
Final release 1.0.0

### ISSUE THREAD

**Issue by mxochicale** [SciKit-Surgery/scikit-surgerybard#115]
I am opening this issue to prepare the final release 2.0.0 of  scikit-surgerybard. 

* [x] Review issues (check status, decide if we address them or leave them for future work, etc) 
* [x] add label priorities to issues (p0-critical, p1-high, p2-medium, p3-low)
* [x] Update package versions in requirements  and perform local and remote unit tests
  * LOCAL UNIT TEST with python3.8 in Ubuntu 22.04.1 LTS ; Linux 5.19.0-45-generic; x86-64 > "36 passed in 10.93s"
  * REMOTE UNIT TEST ubuntu-latest python3.8  > [36 passed in 9.72s](https://github.com/SciKit-Surgery/scikit-surgerybard/actions/runs/5350027944/jobs/9702079589#step:6:716)
  * REMOTE UNIT TEST windows-latest, python3.8 > [36 passed in 22.07s](https://github.com/SciKit-Surgery/scikit-surgerybard/actions/runs/5350027944/jobs/9702079755#step:7:148)
  * REMOTE UNIT TEST macos-latest, python3.8 > [36 passed in 61.78s (0:01:01)](https://github.com/SciKit-Surgery/scikit-surgerybard/actions/runs/5350027944/jobs/9702079907#step:7:135)
* [x] Updates ci.yml for QT settings and Qt for X11 Requirements
* [x] Update classifiers in setup.py https://pypi.org/classifiers/
* [x] Update settings (branches, .readthedocs.yaml) in https://readthedocs.org/dashboard/scikit-surgerybard/advanced/ 
* [x] Fix docs removed `def generate_apidocs(*args):` for fix duplication call of 
* [x] Solved `WARNING: Unexpected indentation.` when sphinx-build
* [x] Solved https://github.com/SciKit-Surgery/scikit-surgerybard/issues/114
* [x] Open discussions https://github.com/SciKit-Surgery/scikit-surgerybard/discussions
* [x] Created release 1.0.0
  git tag -a 'v1.0.0' -m "Release version v1.0.0"
  git push origin v1.0.0
* [x] Published pypi final release 1.0.0 https://pypi.org/project/scikit-surgerybard/#history > https://pypi.org/project/scikit-surgerybard/1.0.0/

**Comment by mxochicale** [comment]
Getting a requirements conflict 
```
The conflict is caused by:
The user requested scikit-surgeryvtk>=2.0.1
scikit-surgeryutils 2.0rc0 depends on scikit-surgeryvtk==2.0rc0
``` 
let's wait for scikit-surgeryutils 2.0.0

**Comment by mxochicale** [comment]
:tada:  Published pypi final release 1.0.0 https://pypi.org/project/scikit-surgerybard/#history > https://pypi.org/project/scikit-surgerybard/1.0.0/
